core/domain/translation_fetchers.py

```diff
@@ -20,12 +20,22 @@
 
 from core.domain import translation_domain
 from core.platform import models
+from core.storage.translation import gae_models
```

Comment:
Why do you import this? Just use `translation_models` that you import below.

Reply:
Done!